packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -215,7 +215,6 @@ const getAllProps = (
     // Visual props
     backgroundColor: getBackgroundValue(props.backgroundColor, props.theme, breakpoint),
     borderRadius: getBorderRadiusValue(props.borderRadius, props.theme, breakpoint),
-    transform: getResponsiveValue(props.transform, breakpoint),
```

Comment:
`transform` was only allowed in BaseBox and not exported to consumer. 

`transform` is very different on web vs native. On Native it takes a completely different object.

I went through our codebase and saw it was also never used. So don't see it's worth adding bunch of checks to make it work on web and native both when its never really used.